codes/classical/properties/LDPC.yml

```diff
@@ -0,0 +1,33 @@
+
+code_id: LDPC
+
+name: 'Low Density Parity Check Codes'
+introduced: '\cite{arXiv:0805.0241}'
+
+
+Description: |
```

Comment:
This need substantial revision. E.g., the "low density" part is not mentioned, which is key to the defn. The ref is not the correct one for LDPC code. Previous comments are not addressed.